src/app/home/navigation.component.ts

```diff
@@ -11,7 +11,17 @@ import { Router } from '@angular/router';
       <li><a href="#" class="km-logout" (click)="logoutClick()">LOGOUT</a></li>
       <li><a routerLink="/manager"><i class="material-icons">settings</i></a></li>
       <li *ngIf="roles.indexOf('_admin') === -1"><a routerLink="/users/profile/{{name}}"><mat-icon>person</mat-icon></a></li>
+      <li><a href="#"><mat-icon [matMenuTriggerFor]="notification">notifications</mat-icon></a></li>

```

Comment:
When clicked acts as a link to `/#`.  Please remove the link as this should simply open the menu.